mscr-ui/src/common/interfaces/crosswalk.interface.ts

```diff
@@ -5,13 +5,15 @@ import { ContentRevision } from '@app/common/interfaces/content-revision.interfa
 import { Metadata } from '@app/common/interfaces/metadata.interface';
 import { Format } from '@app/common/interfaces/format.interface';
 import { Visibility } from '@app/common/interfaces/search.interface';
+import { Organization } from './organizations.interface';
 
 export interface Crosswalk extends Metadata {
   status?: string | undefined;
   languages?: string[];
   organizations?: string[];
   sourceSchema: string;
   targetSchema: string;
+  owner?: string[]; // Added owner for checking permission
```

Comment:
Owner property could have been added to Metadata interface, which both Schema and Crosswalk extend